src/topology/basic.lean

```diff
@@ -569,6 +569,23 @@ lemma mem_closure_of_tendsto {f : β → α} {b : filter β} {a : α} {s : set 
 mem_of_closed_of_tendsto hb hf (is_closed_closure) $
   filter.mem_sets_of_superset h (preimage_mono subset_closure)
 
+/-- If `f` sends the complement to `s` to a single point `a`, then for any filter `l` we have
+`tendsto f (l ⊓ principal s) (𝓝 a) ↔ tendsto f l (𝓝 a)`. -/
```

Comment:
repeating verbatim the conclusion of the lemma is not very useful. What about "then for any filter `l` it is equivalent to converge to `a` along `l` restricted to `s`, and to converge to `a` along `l`"?

Reply:
Done.